Gruntfile.js

```diff
@@ -27,7 +27,7 @@ module.exports = function(grunt) {
       header: '+function ($) {\n' +
               '  "use strict";\n' +
               '\n' +
-              '  if (!("ontouchstart" in window || navigator.msMaxTouchPoints)) return false \n\n',
+              '  if (!("ontouchstart" in window || "PointerEvent" in window || navigator.msMaxTouchPoints)) return false \n\n',
```

Comment:
@patrickkettner Oh dam it...didn't realize that! So all my comments below are obsolet. Sorry man :v: